### DISCUSSION TITLE
No WiFi SSID

### DISCUSSION THREAD

**Discussion by uncleiven** [tesla-android/issue-tracker#187] [Q&A] [answered]
Hi have loaded 2022.45.1 onto an SD-card 3 times now yet when I power up the Pi, there is no WiFi SSID being broadcasted. Any ideas?

**Comment by uncleiven** [comment, marked as answer]
Managed to get it working now

**Reply by kirkmfreeman** [reply]
How did you get it working?  I am having the same issue.  No SSID is being broadcast.

**Reply by mikegapinski** [reply]
My guess? USB cable or weak power source ;)

**Reply by kirkmfreeman** [reply]
Thanks!  Will try a separate power adapter.

**Reply by mikegapinski** [reply]
It has to work with the USB-C ports in 2021+ cars if you have a good cable that supports USB-C PD. 

For older cars I would recommend using a 12V power supply with USB-C PD.

**Reply by kirkmfreeman** [reply]
What is the SSID it will broadcast?  Different per device?

**Reply by mikegapinski** [reply]
Tesla-Android_xxxx

**Reply by kirkmfreeman** [reply]
Thanks again.
